Src/SDK/WorkflowClient/Temporal.WorkflowClient.Errors/public/CancellationException.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Temporal.Api.Failure.V1;
+using Temporal.Common.Payloads;
+using Temporal.Serialization;
+
+namespace Temporal.WorkflowClient.Errors
+{
+    public sealed class CancellationException : Exception, ITemporalFailure
```

Comment:
```suggestion
    public sealed class CancelledException : Exception, ITemporalFailure
```

Let's stay closer to the proto names and maintain consistency with other SDKs.
Also, `CancellationException` sounds like the act of cancellation was not successful.